### ISSUE TITLE
vssadmin delete shadows /All /Quiet

### ISSUE THREAD

**Issue by gitnol** [semazurek/ET-All-in-One-Optimizer#10]
Hi,
It is not needed / there is no reason to delete all shadow copies.
IMHO: So please remove it, if possible...
best regards
Markus

**Comment by semazurek** [comment]
Good point ! I immediately removed, corrected and created a separateg backup module inside :) 
I need suggestions like these <3
